src/include/simeng/Elf.hh

```diff
@@ -25,6 +25,7 @@ struct ElfHeader {
 class Elf {
  public:
   Elf(std::string path);
```

Comment:
Why are we keeping this constructor?

Reply:
I have left most of the old code, comments, etc because I wanted you and Finn you approve the new methodology first. Will be tidying it up soon